libcamera-sys/c_api/camera_manager.cpp

```diff
@@ -33,6 +34,7 @@ libcamera_camera_t *libcamera_camera_manager_get_id(libcamera_camera_manager_t *
         return new libcamera_camera_t(camera);
 }
 
+#if LIBCAMERA_VERSION_MAJOR == 0 && LIBCAMERA_VERSION_MINOR == 0
 libcamera_camera_t *libcamera_camera_manager_get_dev(libcamera_camera_manager_t *mgr, dev_t dev) {
```

Comment:
get_dev(... dev_t dev) was never supposed to be used externally or be public.

You should probably just remove this function entirely and replace it with the one below. The mgr->get(const std::string id); has been valid in all releases.

Reply:
I removed the function. I did not need it, it's just not compatible.